cmd/cookiemonster/main.go

```diff
@@ -29,13 +30,19 @@ func main() {
 	flag.Parse()
 
 	// We need both of these.
-	if (*cookieFlag == "" && *urlFlag == "") || (*cookieFlag != "" && *urlFlag != "") || *wordlistFlag == "" {
+	if (*cookieFlag == "" && *urlFlag == "" && *fileFlag == "") || *wordlistFlag == "" {
```

Comment:
You should preserve the original behavior here and error if 2 or more of `cookieFlag`, `urlFlag`, `fileFlag` are set. Right now you removed the `(*cookieFlag != "" && *urlFlag != "")` conditional.

Reply:
I understand but if done using if/else, adding a new flag requires checking more combinations which becomes a little complex, in my opinion. For example, it would become something like;
```go
// if wordlist is empty
if (*wordlistFlag == "") {
  flag.Usage()
  os.Exit(1)
}
// if none of the flags are set
if (*cookieFlag == "" && *urlFlag == "" && *fileFlag == "") {
  flag.Usage()
  os.Exit(1)
}
// if either of 2 out of 3 flags are set
if (*cookieFlag != "" && *urlFlag != "") || (*cookieFlag != "" && *fileFlag != "") || (*url != "" && *fileFlag != "") {
  flag.Usage()
  os.Exit(1)
}
```

That doesn't seem necessary if we prioritise one flag over another. That's why I added comments in lines 38-39. Though, it should be added in the docs if we go that route.